components/configuration-mgt/org.wso2.carbon.identity.configuration.mgt.core/src/main/java/org/wso2/carbon/identity/configuration/mgt/core/constant/ConfigurationConstants.java

```diff
@@ -0,0 +1,117 @@
+        ERROR_CODE_RESOURCE_ADD_REQUEST_INVALID("CONFIGM_00010", "Resource add request validation failed. " +
+                "Invalid resource add request."),
+        ERROR_CODE_RESOURCE_GET_REQUEST_INVALID("CONFIGM_00011", "Resource get request validation failed"),
+        ERROR_CODE_GET_RESOURCE("CONFIGM_00012", "Error while getting the resource : %s."),
+        ERROR_CODE_RESOURCE_ALREADY_EXISTS("CONFIGM_00013", "Resource with the name: %s already exists."),
+        ERROR_CODE_ADD_RESOURCE("CONFIGM_00014", "Error while adding the resource : %s."),
+        ERROR_CODE_QUERY_LENGTH_EXCEEDED("CONFIGM_00015", "Too large SQL query length."),
+        ERROR_CODE_RESOURCE_DELETE_REQUEST_REQUIRED("CONFIGM_00016", "Resource delete request validation failed. " +
+                "Invalid resource delete request."),
+        ERROR_CODE_RESOURCE_DOES_NOT_EXISTS("CONFIGM_00017", "Resource with the name: %s does not exists."),
+        ERROR_CODE_SEARCH_REQUEST_INVALID("CONFIGM_00018", "Search request validation failed. " +
+                "Invalid search filter."),
+        ERROR_CODE_SEARCH_TENANT_RESOURCES("CONFIGM_00019", "Error occurred while searching for resources."),
+        ERROR_CODE_RESOURCES_DOES_NOT_EXISTS("CONFIGM_00020", "Resources does not exists."),
+        ERROR_CODE_SEARCH_QUERY_PROPERTY_DOES_NOT_EXISTS("CONFIGM_00021", "Search query property: %s is either " +
+                "invalid or not " +
+                "found in the permitted properties."),
+        ERROR_CODE_SEARCH_QUERY_SQL_PROPERTY_PARSE_ERROR("CONFIGM_00022", "Search query syntax error in the " +
+                "condition: %s."),
+        ERROR_CODE_SEARCH_QUERY_SQL_PARSE_ERROR("CONFIGM_00023", "Search query syntax error"),
+        ERROR_CODE_ATTRIBUTE_IDENTIFIERS_REQUIRED("CONFIGM_00024", "One or more identifiers for the attribute: %s " +
+                "validation failed."),
+        ERROR_CODE_GET_ATTRIBUTE("CONFIGM_00025", "Error while replacing the attribute: %s."),
+        ERROR_CODE_ATTRIBUTE_REQUIRED("CONFIGM_00026", "Attribute validation failed"),
+        ERROR_CODE_ATTRIBUTE_DOES_NOT_EXISTS("CONFIGM_00027", "Attribute with the key: %s does not exists."),
+        ERROR_CODE_UPDATE_ATTRIBUTE("CONFIGM_00028", "Error while updating the attribute: %s."),
+        ERROR_CODE_DELETE_ATTRIBUTE("CONFIGM_00029", "Error while deleting the attribute: %s."),
+        ERROR_CODE_UPDATE_RESOURCE_TYPE("CONFIGM_00030", "Error while updating the resource type: %s."),
+        ERROR_CODE_ATTRIBUTE_ALREADY_EXISTS("CONFIGM_00031", "Attribute with the name: %s already exists."),
+        ERROR_CODE_INSERT_ATTRIBUTE("CONFIGM_00032", "Error while adding the attribute: %s."),
+        ERROR_CODE_RESOURCE_REPLACE_REQUEST_INVALID("CONFIGM_00033", "Resource replace request validation failed."),
+        ERROR_CODE_RESOURCE_REPLACE_REQUEST_INVALID_PARTIAL_UPDATE("CONFIGM_00034", "Resource replace request " +
```

Comment:
both has the same error code CONFIGM_00034

Reply:
Fixed 99bb001a3a26744f49d1944bc07589e267d11a05